include/zephyr/net/wifi_mgmt.h

```diff
@@ -415,6 +422,21 @@ struct wifi_raw_scan_result {
 	uint8_t data[CONFIG_WIFI_MGMT_RAW_SCAN_RESULT_LENGTH];
 };
 #endif /* CONFIG_WIFI_MGMT_RAW_SCAN_RESULTS */
+
+/** Wi-Fi promiscuous mode setup */
+struct wifi_promisc_setup {
+	union {
+		/** promiscuous mode settings */
```

Comment:
Please begin with uppercase, here and below.

Reply:
will make the relevant change